keyboards/lily58/keymaps/encoder/keymap.c

```diff
@@ -0,0 +1,231 @@
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
```

Comment:
```suggestion
```
Fairly sure this block isn't needed.

Reply:
It's in the [original default keymap](https://github.com/qmk/qmk_firmware/blob/master/keyboards/lily58/keymaps/default/keymap.c#L3). If it should be removed from there, then I feel like that should be batched into a separate PR instead of in this one.

Reply:
It should be removed there, as well, but that would be a separate PR, ideally.  

As in, for all the default keymaps that do this.  

However, replicating the "bad code" should be avoided, as well.